.vsts-ci.yml

```diff
@@ -278,4 +282,5 @@ jobs:
 #         - Darwin
 #         - Copy_SDK_To_Latest
 #       queue:
-#         name: Hosted VS2017
\ No newline at end of file
+#         name: Hosted VS2017
```

Comment:
Can we just delete this commented out part? I added time out to it as well. In case we uncomment it later